docs/parset.rst

```diff
@@ -185,340 +227,405 @@ This computes the median absolute residual (i.e. median absolute deviation from
+    unless OUTNAME contains a slash, in which case OUTDIR is ignored and OUTNAME is
+    taken to be a full base path. Default: cc.
+--out-overwrite=bool
+    Allow overwriting of existing output files. If this is
+    set, and the output parset file exists, will raise an exception.
+    Default: False.
+--out-backup=bool
+    Allow automatic backup of existing output directories. Automatic backup
+    is only used when OUTDIR is used (i.e. OUTNAME doesn't contain any slashes),
+    and it ends with .cc-out (implicitly or explicitly). In this case, existing
+    output directories are renamed to .cc.out.0, .1, etc. Default: True.
+--out-mode=keyword
+    Operational mode. [so] solve only; [sc] solve and
+    generate corrected visibilities; [sr] solve and
+    generate corrected residuals; [ss] solve and generate
+    uncorrected residuals; [ac] apply solutions, generate
+    corrected visibilities; [ar] apply solutions, generate
+    corrected residuals; [as] apply solutions, generate
+    uncorrected residuals. Keywords: so, sc, sr, ss, ac,
+    ar, as. Default: sc.
+--out-apply-solver-flags=bool
+    Apply solver flags when writing new data to measurement set.
+    Default: True.
+--out-derotate=multi
+    Explicitly enables or disables derotation of output visibilities.
+    Default (None) is to use the --model-pa-rotate and --model-feed-rotate
+    settings. Options: None | 0 | 1.
 --out-column=str
-	Output MS column name (if applicable). Default: CORRECTED_DATA.
+    Output MS column name (if applicable). Default: CORRECTED_DATA.
 --out-model-column=str
-	If set, model visibilities will be written to the
-	specified column. No default.
+    If set, model visibilities will be written to the
+    specified column. No default.
+--out-weight-column=str
+    If set, weights from the Robust Solver will be written to the specified column.
+    This should be set only if we are using the robust solver.
+    No default.
 --out-reinit-column=bool
-	Reinitialize output MS column. Useful if the column is
-	in a half-filled or corrupt state. Default: 0.
+    Reinitialize output MS column. Useful if the column is
+    in a half-filled or corrupt state. Default: 0.
 --out-subtract-model=int
-	Index of model to subtract, if generating residuals.
-	Default: 0.
+    Index of model to subtract, if generating residuals.
+    Default: 0.
 --out-subtract-dirs=multi
-	Which model directions to subtract, if generating
-	residuals. ":" subtracts all. Can also be specified as
-	"N", "N:M", ":N", "N:", "N,M,K". Default: :.
---out-plots=bool     
-	Generate summary plots. Default: 1.
+    Which model directions to subtract, if generating
+    residuals. ":" subtracts all. Can also be specified as
+    "N", "N:M", ":N", "N:", "N,M,K". Default: :.
+--out-plots=bool
+    Generate summary plots. Default: 1.
 --out-plots-show=bool
-	Show summary plots interactively. Default: 0.
+    Show summary plots interactively. Default: 0.
 --out-casa-gaintables=bool
```

Comment:
we should remove this option. Python casacore 3 no longer supports writing the necessary metadata to these tables.

Reply:
I will relegate this change to a future PR - currently we print warnings everywhere anyway. This PR only fixes the docs to be more consistent with the -h option.